.github/workflows/demo_apps_build.yaml

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build example - Telink
+
+on:
+    push:
+
+concurrency:
+    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
+    cancel-in-progress: true
+
+env:
+    CHIP_NO_LOG_TIMESTAMPS: true
+
+jobs:
+    telink:
+        name: Telink
+        env:
+            BUILD_TYPE: telink
+            ZEPHYR_TOOLCHAIN_VARIANT: zephyr
+            ZEPHYR_SDK_VERSION: 0.15.2
+            ZEPHYR_SDK_INSTALL_DIR: zephyr_sdk/zephyr-sdk-"${ZEPHYR_SDK_VERSION}"
+
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: .
+        if: github.actor != 'restyled-io[bot]'
+
+        steps:
+            - uses: Wandalen/wretry.action@v1.0.36
+              name: Checkout
+              with:
+                  action: actions/checkout@v3
+                  with: |
+                      token: ${{ github.token }}
+                  attempt_limit: 3
+                  attempt_delay: 2000
+
+            - name: Move all files to zephyr directory
```

Comment:
What is the reason of this?
What is inside zephyr.zip and why we need to use it?
Why we can't use Zephyr sources directly without some manipulations?

Reply:
We are using zephyr sources files, but I only put them into zephyr directory, this zephyr.zip file is removed at the end, it is needed only for moving files for creating the same directory structure as we have on local systems. Then it is much more clear for me how to move in the repo.

I think it can be changed, but if it is disturbing for us ? .. it takes only 10 seconds additional time for the overall process ..

Anyway, I removed it.